index.html

```diff
@@ -1,133 +1,175 @@
-        <div class="name">
-          Matthew Jones
-          <span class="pledged">$50</span>
-        </div>
-      </div>
-      <div class="supporter">
-        <div class="name">
-          Ryker Hatch
-          <span class="pledged">$10</span>
-        </div>
-      </div>
-      <div class="supporter">
-        <div class="name">
-          Randy Mills
-          <span class="pledged">$10</span>
-        </div>
-      </div>
-      <div class="supporter">
-        <div class="name">
-          Silvio Verzilli
-          <span class="pledged">$20</span>
-        </div>
-      </div>
-      <div class="supporter">
-        <div class="name">
-          Andy Alfred
-          <span class="pledged">$10</span>
-        </div>
-      </div>
-      <div class="supporter">
-        <div class="name">
-          Mike Stewart
-          <span class="pledged">$12.66</span>
-        </div>
-      </div>
+  <body>
+    <div style="text-align: center">
+      <img src="https://media.giphy.com/media/l2JhwdnrGvfnoXrzi/giphy.gif" width="200px" />
     </div>
-  </div>
-
-  <div class="contact">
-    Please contact: <b>@josh.northrup</b> on Slack
-  </div>
-</body>
+    <h1 style="text-align: center">Introhive 3D Printer Fund</h1>
+    <hr />
+    <h3>Executive Summary</h3>
+    <ul>
+      <li>
+        Target: <strong>$312.66</strong>
+      </li>
+      <li style="color: green">
+        We have: <strong style="color: green">$192.66</strong>
+      </li>
+      <li style="color: red">
+        We need: <strong style="color: red">$120.00</strong>
+      </li>
+      <li>
+        We do <strong>NOT</strong> have enough for a 3D Printer 😔
+      </li>
+    </ul>
+    <hr style="border-top: 1px solid #666" />
+    <h3>Current Supporters:</h3>
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Pledge</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            Josh Northrup
+          </td>
+          <td>
+            $20
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Alex Inman
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Matthew Cassidy
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Stephen Carr
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Jason Gillis
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Stephen Downward
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Kalpana Challa
+          </td>
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Matthew Jones
+          </td>
+          <td>
+            $10
```

Comment:
@mdonjones 😱

Reply:
I think we need a test suite to avoid these unintended changes